types/commander/index.d.ts

```diff
@@ -17,7 +16,7 @@ declare namespace commander {
         unknown: string[];
     }
 
-    class Option {
+    interface Option {
```

Comment:
How are you suggesting to change it? If I can it to a class. I'll get
```
[ts] Duplicate identifier "commander"
```

And if I do the class outside the `namespace`,  and export as

```
declare const commander: commander.StaticCommander & Option;
```

I can still do 

```
new(new program.Option("--f"))("--f")
```